comp/trace/config/setup.go

```diff
@@ -194,7 +194,9 @@ func applyDatadogConfig(c *config.AgentConfig, core corecompcfg.Component) error
 		c.SkipSSLValidation = core.GetBool("skip_ssl_validation")
 	}
 	if core.IsSet("apm_config.enabled") {
-		c.Enabled = core.GetBool("apm_config.enabled")
+		isAPMEnabled := (core.GetBool("apm_config.enabled") && utils.IsCoreAgentEnabled(core)) ||
+			core.GetBool("apm_config.error_tracking_standalone.enabled")
+		c.Enabled = isAPMEnabled
```

Comment:
Why does the trace-agent now require this extra logic to determine if it's enabled when an explicit setting exists? 🤔 
Isn't `apm_config.enabled` a requisite for ETS?

Reply:
Thank you. The doc doesn't mention that `error_tracking_standalone.enabled` is nested under `apm_config`.  If the customer is configuring ETS under `apm_config`, I would expect them to also `enable` it? 🤔

Reply:
> The doc doesn't mention that error_tracking_standalone.enabled is nested under apm_config

That's missing in the doc, good catch. Fixing it there as it's supposed to be under `apm_config`

> If the customer is configuring ETS under apm_config, I would expect them to also enable it?

No, the goal was to have the minimum number of options that need to be enabled by the user. The best agreement we found w/ PMs on the agent side was to have only  `apm_config.error_tracking_standalone.enabled` and not require `apm_config.enabled` to have fewer settings to change to enable the functionality.